src/main/java/us/ilite/robot/modules/VioletDriveModule.java

```diff
@@ -230,6 +245,10 @@ public void readInputs() {
         db.imu.set(EGyro.HEADING_DEGREES, -mGyro.getHeading().getDegrees());
         db.imu.set(EGyro.YAW_OMEGA_DEGREES, mGyro.getYawRate().getDegrees());
 
+        mOdometry.update(
+                mGyro.getHeading(), mLeftEncoder.getPosition() * kDriveNEOPositionFactor * 0.348,
+                mRightEncoder.getPosition() * kDriveNEOPositionFactor * 0.348);
```

Comment:
What are these magic numbers? Recommend moving them to one of the constants files with proper documentation